package.json

```diff
@@ -3,6 +3,8 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "google-maps-react": "^2.0.2",
```

Comment:
_Suggestion_: Remove dependencies that you're not using

Reply:
*sigh* Removed!